src/buildercore/lifecycle.py

```diff
@@ -123,7 +123,10 @@ def start(stackname):
 
     try:
         wait_for_ec2_steady_state(stackname, ec2_to_be_checked)
-    except EC2Timeout:
+    except EC2Timeout as e:
```

Comment:
Checking an error string doesn't seem like a good idea. Typically you would create an exception for the specific problem.

Also `e` is not very descriptive. Perhaps rename the variable to `error` or `exception`?

Reply:
`except EC2Timeout as exception` ... really?

Reply:
Yes. `exception`, in this case, is a variable with a name that describes the object it represents, which in this case, is an Exception class object. You could go one step further and provide a variable name which describes the type of exception. The point being that when someone reads the code further down the script, it's easy for the reader to know what that variable contains. Using single character variables doesn't provide this and is generally considered a bad practice.

Reply:
Explained in person the error string, which stems from the fact that it's Fabric raising the exception without using more specific types that we can catch.

Reply:
No opinions on the variable name, so by default leaving it consistent with the rest of the project and separating that decision if we want to make them more readable.